build/_k-test.shade

```diff
@@ -3,6 +3,7 @@ use import="Environment"
 
 default NO_PARALLEL_TEST_PROJECTS='${E("NO_PARALLEL_TEST_PROJECTS")}'
 default KOREBUILD_TEST_DNXCORE='${E("KOREBUILD_TEST_DNXCORE")}'
+default CORECLR_RUNTIME_VERSION='${E("CORECLR_RUNTIME_VERSION")}'
```

Comment:
Suggest this environment variable name is too specific for KoreBuild. Name should mention testing and align with `KOREBUILD_TEST_DNXCORE`. How about `KOREBUILD_TEST_DNXCORE_VERSION`?

Also seems strange to support this override only for DNX Core testing. Should CLR / Mono testing call `Dnx()` with a `dnvmUse` parameter when e.g. `KOREBUILD_TEST_VERSION` is non-empty?